### ISSUE TITLE
Maintenance plan (please read)

### ISSUE THREAD

**Issue by rviscomi** [HTTPArchive/almanac.httparchive.org#488]
Congratulations to everyone on a successful launch! Our [analytics](https://datastudio.google.com/s/osyqyOgNUOg) are looking great with over 40k page views in the three days since launch. I just wanted to take this time to discuss the maintenance plan for the Almanac now that we've reached this big milestone.

There are still a couple of P0 action items that must be resolved by the end of the week. A few chapters are marked "unedited" and we need to get these edited and visualized for parity with the rest of the chapters. See #218 and #237 for the status of each chapter.

There are also a few open issues in the [Ship it!](https://github.com/HTTPArchive/almanac.httparchive.org/milestone/5) milestone which would be great to close out ASAP. The [Après Ski](https://github.com/HTTPArchive/almanac.httparchive.org/milestone/6) milestone contains lower priority tasks that should get done in the next few weeks.

After such a hard push to get this launched, I think everyone deserves a bit of a break (myself especially). I've been planning to take the month of December entirely off from all work-related activities. During this time, and given the holidays, I think it'd be best to freeze the production website and refrain from deploying any changes live, barring any emergencies.

For anyone who is still eager to squash bugs and build new features, I won't stop you! I ask that you continue our issue and PR conventions:

- create a new issue for all bugs and feature requests
- open a forum for discussion and try to reach a consensus about a solution
- file a PR to resolve the issue and get the changes reviewed by at least one other person
- only after approval, merge the changes into the `master` branch

Everyone who is on the Almanac Contributors team or subteams has edit access to the repository. New contributors will need someone from this team to merge their PRs. I trust the wisdom of @obto @bazzadp and @mikegeyser to help guide the day-to-day project in my absence.

All that being said, we still have some time before December. Let's try to get the Almanac into a stable state that we would feel good about not touching for a month. This means resolving any lingering SEO (#286), accessibility (#303), or performance (#413) issues. I would also love to take this time to focus on translations (#38).

One of the themes I'm seeing in all of the warm reception to the Almanac is the admiration for the community of contributors we've built. I really appreciate everyone's hard work, trust, cooperation, and expertise getting us to this point and continuing to move the Almanac forward. I hope you all would [consider participating](https://forms.gle/K5C2TG322BBvXfYk8) in the 2020 Almanac next year!

**Comment by AymenLoukil** [comment]
@rviscomi the links on SEO, accessibility and performance are not ok :)

**Comment by bazzadp** [comment]
> @rviscomi the links on SEO, accessibility and performance are not ok :)

Fixed!

**Comment by ymschaap** [comment]
Enjoy your break!

**Comment by bazzadp** [comment]
Hey @rviscomi could you give advice on how to tag issues correctly while you're out?

I think the Labels are fairly obvious and will try to tag "Good First Issue" as much as possible where appropriate (and also give newbies a chance to answer some of these!! 😀). Let us know any other tips or ways you are using them.

On projects I can't seem to tag Web Almanac-TODO or any other status. Need to permission me for those?

On milestones I presume [SHIP-IT milestone](https://github.com/HTTPArchive/almanac.httparchive.org/milestone/5) shouldn't really be used for new issues? Is [Après-Ski](https://github.com/HTTPArchive/almanac.httparchive.org/milestone/6) for those we hope to close out before change freeze? Do we need a third milestone for after that?

**Comment by bazzadp** [comment]
Hey @rviscomi , I presume you're off for Thanksgiving (Happy Thanksgiving btw!).

Not sure if you're started your break or will be back on this next week? If you see this message then let us know one way or another.

> On projects I can't seem to tag Web Almanac-TODO or any other status. Need to permission me for those?

This seems to be fixed for me so thanks!

We still can't invite people to Translators team which I think would be helpful because there's a fair bit going on there and would be good to be able to reach out to people.

There's also [quite a few PRs in the queue](https://github.com/HTTPArchive/almanac.httparchive.org/pulls) at the moment. I'd like to merge some of those and do a final release before we lock it down, to get the print view working and also set us up better for translations, as suspect we'll make some good progress on a few of those over the freeze.

Anyway let us know your status if you see this message and then I'll promise to give you a break until the new year 😀Thanks again for all your efforts on this!

**Comment by rviscomi** [comment]
### Labels 
Label liberally. "Good first issue" for anything trivially simple or something you're willing to mentor new contributors on.

### Project
Yes, everything in this repo should have the Almanac Project attached. Update the status between TODO/Needs Attention/In Progress/Done accordingly. GH should move things automatically in some cases like closing PRs.

### Milestone
You're correct. No need for new milestones as we're basically in cruise control until the 2020 edition.

### Teams
The team thing is a broken UX, and it's annoyingly difficult to add new members. There IS a way for a newcomer to request membership, but I don't have a non-member GitHub account to test that flow. Team pages are behind some kind of membership authentication so sharing those links doesn't work. Ideally let's figure out how to allow people to request membership. Can't figure out why I can't delegate invitations to you @bazzadp.

### Outstanding PRs
I've done reviews on all of the non-translation PRs. Feel free to deploy when you're ready.

### My status
My hiatus is starting today and I'll be changing my repo notifications from "all" to "mention only" to reduce distractions but I'm still here if anything comes up. Just @ me.

**Comment by bazzadp** [comment]
All deployed. Let's call that a freeze unless we see a production issue.

**Comment by rviscomi** [comment]
I'm back!

**Comment by bazzadp** [comment]
> I'm back!

Woo hoo! Welcome back and hope you got enough of a break.
